cmd/keycloakb/keycloak_bridge.go

```diff
@@ -1378,6 +1387,10 @@ func config(ctx context.Context, logger log.Logger) *viper.Viper {
 	v.BindEnv(CfgValidationBasicAuthToken, "CT_BRIDGE_VALIDATION_BASIC_AUTH")
 	censoredParameters[CfgValidationBasicAuthToken] = true
 
+	v.BindEnv(CfgDbAesGcmKey, "CT_BRIDGE_DB_AES_KEY")
+	censoredParameters[CfgDbAesGcmKey] = true
+	v.SetDefault(CfgDbAesGcmTagSize, 16)
```

Comment:
Would be nice to move setDefault with all the others

Reply:
DONE